otelcollector/telegraf/telegraf-prometheus-collector.conf

```diff
@@ -118,6 +118,23 @@
   [processors.converter.fields]
     float = ["*"]
 
+# Transforms tag and field values as well as measurement, tag and field names with regex pattern
+[[processors.regex]]
+  namepass = ["target_allocator"]
+
+  # Tag and field conversions defined in a separate sub-tables
+  [[processors.regex.tags]]
+    ## Tag to change, "*" will change every tag
+    key = "job_name"
+    ## Regular expression to match on a tag value
+    pattern = '(^(podMonitor|serviceMonitor)\/.*)|(.*)'
```

Comment:
nit: could you pls also add a comment with what the generated podmonitor and servicemonitor names look like?

Reply:
Yes sounds good, I'll commit this to the operator-targetallocator branch